packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.html

```diff
@@ -14,8 +14,8 @@
         <div class="slds-text-body_small empty-list-message slds-p-horizontal_medium slds-var-m-vertical_small">{labels.emptyListLabel}</div>
       </template>
       <ul>
-        <template for:each={queries} for:item="query">
-          <li class="slds-var-m-vertical_x-small slds-grid" key={query}>
+        <template for:each={queries} for:item="query" for:index="index">
+          <li key={query} value={index} onclick={executeQuery} class="slds-var-m-vertical_x-small slds-grid">
             <div class="slds-col recent-query__container">
```

Comment:
Just noticing that now, but I believe it would be a better idea to use the anchor html element for those I think:
https://www.lightningdesignsystem.com/utilities/interactions/#Link-Reset-with-Text-Link-Inside
You should inherit already from the hover styles and branding from the community most likely.

Reply:
On using the anchor element, there are a few issues I am facing. The first one is the value shows as undefined in the executeQuery for which I did get a solution, but the solution disables the pointer and the underline on hover. So i have to again add li:hover in css


`<li key={query} value={index} onclick={executeQuery} class="slds-var-m-vertical_x-small slds-grid">

            <a href="#" class="slds-col recent-query__container">

              <lightning-icon class="recent-query__icon slds-current-color" icon-name="utility:search" alternative-text="search icon" size="xx-small"></lightning-icon>

              <span class="slds-text-body_small nowrap query-text__container slds-truncate slds-text-link">
               {query}
              </span>

            </a>

          </li>
`
the css includes:
 li a {pointer-events:none;}

Reply:
mmm yes i agree. would be easier to keep it consistent with the recent-results-list which uses `<a>` tags as well

Reply:
Yea, I believe this is a problem `li a {pointer-events:none;}` and that you should override it by adding a class into it.

`li a.my-class {pointer-events:auto;}`